modules/aws/master-asg/master.tf

```diff
@@ -32,6 +32,8 @@ resource "aws_autoscaling_group" "masters" {
   min_size             = "${var.instance_count}"
   launch_configuration = "${aws_launch_configuration.master_conf.id}"
   vpc_zone_identifier  = ["${var.subnet_ids}"]
+  metrics_granularity  = "1Minute"
```

Comment:
Default is 1Minute: https://www.terraform.io/docs/providers/aws/r/autoscaling_group.html#metrics_granularity

I presume we want to be declarative here, right?

Reply:
Yes, it's just to be declarative, and in the future  in they change we will have that value and dashboards and alarms will continue  to work the same way

Reply:
👍 Nice!